### ISSUE TITLE
whenOpenedDo: is based on isClosed property and it looks strange

### ISSUE THREAD

**Issue by Ducasse** [pharo-spec/Spec#1208]
```
whenOpenedDo: aBlock
	"Inform when window is opened. 
	 `aBlock` receives zero arguments."

	self 
		property: #isClosed 
		whenChangedDo: [ :value | value ifFalse: aBlock ]
```

**Comment by Ducasse** [comment]
registering these two methods.... breaks no window is even opened.

```
initializeWindow: aWindowPresenter

	aWindowPresenter whenOpenedDo: [ UIManager default inform: 'When opened' ].
	aWindowPresenter whenClosedDo: [ UIManager default inform: 'When closed' ].
```

**Comment by estebanlm** [comment]
> ```
> whenOpenedDo: aBlock
> 	"Inform when window is opened. 
> 	 `aBlock` receives zero arguments."
> 
> 	self 
> 		property: #isClosed 
> 		whenChangedDo: [ :value | value ifFalse: aBlock ]
> ```

yeah, it looks like a hack to trigger an event (I thought same a couple of times I have visited there), but it is there since a lot and it is working so I did not touch it :P

**Comment by estebanlm** [comment]
> registering these two methods.... breaks no window is even opened.
> 
> ```
> initializeWindow: aWindowPresenter
> 
> 	aWindowPresenter whenOpenedDo: [ UIManager default inform: 'When opened' ].
> 	aWindowPresenter whenClosedDo: [ UIManager default inform: 'When closed' ].
> ```

I was unable to reproduce this (I implemented same in an example and events are being triggered)
